### DISCUSSION TITLE
Abandoning TensorFlow Lite?

### DISCUSSION THREAD

**Discussion by Son-kun** [deepjavalibrary/djl#3216] [Q&A]
Hi, I try to work with DJL and Google-Coral on RPi4, for that I need to switch from PyTorch to TFLite.
After some time I reached the wall... From version 0.28.0 TFLite dissapear from DJL repository. When I try to use 0.27.0 on my app I get error:
`
Caused by: java.io.FileNotFoundException: https://publish.djl.ai/tflite/2.4.1/files.txt
`
Can somebody help me with that? I'm doing something wrong, DJL is abandoning TFLite, or maybe there is another problem?

**Comment by frankfliu** [comment]
In DJL 0.27.0, it should use TFLite 2.6.2, see: https://github.com/deepjavalibrary/djl/blob/v0.27.0/gradle.properties#L18, and you should use https://mvnrepository.com/artifact/ai.djl.tflite/tflite-native-cpu/2.6.2

I guess you are using this dependency in your project: https://mvnrepository.com/artifact/ai.djl.tflite/tflite-native-auto, which has been deprecated long time ago.

We dropped tflite support since 0.28.0:
1. The adoption of TFLite is really low
2. For mobile use case, most customer use onnxruntime

Can you provide why onxxruntime doesn't work for you?

**Reply by Son-kun** [reply]
After some work, I sucessfully run my app & model using onnxruntime on rpi. But still it don't use GoogleCoral TPU.
Is there any way to force djl onnx to work with GC TPU, or any other model on RPi with it?
